install-dependencies/action.yaml

```diff
@@ -73,16 +73,3 @@ runs:
       run: npm ci
       env:
         NPM_AUTH_TOKEN: ${{ inputs.npm-auth-token }}
-    - name: Cache node_modules
-      if: inputs.s3-bucket-name != '' &&  steps.read_cache.outputs.cache-hit != 'true'
-      uses: everpcpc/actions-cache@v2
-      id: write_cache
-      with:
-        bucket: ${{ inputs.s3-bucket-name }}
-        use-fallback: false
-        path: node_modules
-        key: ${{ env.cache-name }}-${{ steps.lockfile_hash.outputs.hash }}
-        restore-keys: ${{ env.cache-name }}-
-      env:
-        AWS_REGION: ${{ inputs.s3-bucket-region }}
-        cache-name: ${{ github.event.repository.name }}/cache-node-modules
```

Comment:
but this is the run that actually pushes to the cache if the entry wasn't there

Reply:
This is not needed

Reply:
The action does it automatically

Reply:
And they don't mention in their docs the need to have a second job to write to the cache